CLAUDE.md

```
@@ -0,0 +1,68 @@
+```bash
+pnpm dev             # Development server
+pnpm build           # Production build (also runs copy-binaries.js postbuild)
+pnpm start           # Start production server
+pnpm lint            # Biome lint
+pnpm knip            # Dead code detection
+pnpm db:deploy       # Run Prisma migrations + generate client
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)